roles/django_app_k8s/templates/rabbitmq.yml.j2

```diff
@@ -0,0 +1,69 @@
+---
+
+kind: Deployment
+apiVersion: v1
+metadata:
+  labels:
+    app.kubernetes.io/name: rabbitmq
+    app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+    app.kubernetes.io/version: "3.7"
+    app.kubernetes.io/component: async-workers
+    app.kubernetes.io/part-of: "{{ django_app_k8s_group }}"
+    app.kubernetes.io/managed-by: Ansible
+spec:
+  # TODO: cluster/HA?
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: rabbitmq
+      app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+      app.kubernetes.io/part-of: "{{ django_app_k8s_group }}"
+  template:
+    metadata:
+      name: opennotificaties-worker
+      labels:
+        app.kubernetes.io/name: rabbitmq
+        app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+        app.kubernetes.io/version: "3.7"
+        app.kubernetes.io/component: async-workers
+        app.kubernetes.io/part-of: "{{ django_app_k8s_group }}"
+        app.kubernetes.io/managed-by: Ansible
+{% for label in django_app_k8s_extra_rabbitmq_labels %}
+        {{ label }}: {{ django_app_k8s_extra_rabbitmq_labels[label] }}
+{% endfor %}                
+    spec:
+      containers:
+      - name: rabbitmq
+        image: rabbitmq:3.7
```

Comment:
maybe a var to control the rabbitmq image/version?

Reply:
added and using: `django_app_k8s_rabbitmq_version: "3.10"`